community/io/src/main/java/org/neo4j/io/pagecache/tracing/cursor/DefaultPageCursorTracerSupplier.java

```diff
@@ -25,7 +25,7 @@
 
     public static final DefaultPageCursorTracerSupplier INSTANCE = new DefaultPageCursorTracerSupplier();
 
-    private DefaultPageCursorTracerSupplier()
+    public DefaultPageCursorTracerSupplier()
```

Comment:
You can just grab the `INSTANCE` where you use this, instead of making the constructor public and creating a new one.

Reply:
Addressed in upcoming commit